Gemfile

```diff
@@ -2,11 +2,18 @@ source 'https://rubygems.org'
 
 ruby '2.2.1'
 
-group :test do
-  gem 'rspec'
+group :development, :test do
+  gem 'bcrypt-ruby'
+  gem 'capybara'
+  gem 'coveralls', require: false
   gem 'cucumber'
-  gem 'rubocop-rspec'
+  gem 'data_mapper'
+  gem 'database_cleaner'
+  gem 'dm-postgres-adapter'
+  gem 'rack-flash3'
+  gem 'rspec'
   gem 'rubocop'
-  gem 'coveralls', require: false
+  gem 'rubocop-rspec'
+  gem 'sinatra'
```

Comment:
Sinatra and DataMapper won't be installed in a production environment! You have them in the development / test block.